src/hatti/constants.cljs

```diff
@@ -4,3 +4,4 @@
 (def _rank "_rank")
 (def _submission_time "_submission_time")
 (def _submitted_by "_submitted_by")
+(def *mapping-threshold* 10000)
```

Comment:
Ok, if we do that it can be by passing configuration options, but I don't
think we need to address this now. When we do let's talk about it and try
to think of the edge cases.